src/arch/riscv/vm.c

```diff
@@ -41,6 +41,12 @@ void vcpu_arch_reset(struct vcpu *vcpu, vaddr_t entry)
     vcpu->regs.a0 = vcpu->arch.hart_id = vcpu->id;
     vcpu->regs.a1 = 0;  // according to sbi it should be the dtb load address
 
+    if (cpu_arch_has_extension(SSTC_EXTENSION)) {
+        CSRW(CSR_STIMECMP, -1);
+        CSRW(CSR_HENVCFG, HENVCFG_STCE);
+    } else {
+        CSRW(CSR_HENVCFG, 0);
```

Comment:
I know we don't have other extensions, but:

```suggestion
        CSRC(CSR_HENVCFG, HENVCFG_STCE);
```